modules/ROOT/pages/salesforce/salesforce-connector-reference.adoc

```diff
@@ -1,3029 +1,4499 @@
+* <<sfdc-config_oauth-saml, OAuth SAML>>
+* <<sfdc-config_oauth-user-pass, OAuth Username Password>>
+ | The connection types that can be provided to this configuration. | |x
+| Expiration Policy a| <<ExpirationPolicy>> | Configures the minimum amount of time that a dynamic configuration instance can remain idle before the runtime considers it eligible for expiration. This does not mean that the platform will expire the instance at the exact moment that it becomes eligible. The runtime actually purges the instances when it sees fit. |  |
+| Fetch All Apex SOAP Metadata a| Boolean | If checked then Datasense is performed for all Apex classes in the organization, otherwise it is performed only for the classes in Apex class names. If the organization contains a lot of Apex classes this might cause ConnectionTimeout during Datasense. Default value is false | false |
+| Fetch All Apex REST Metadata a| Boolean | If checked then Datasense will be performed for all Apex classes in the organization else Datasense will be performed only for the classes in Apex class names. If the organization contains many Apex classes this could cause ConnectionTimeout during Datasense. Default value is false | false |
+| Apex Class Names a| Array of String | List of Apex class names involved in metadata retrieval via Datasense. |  |
 |===
 
-== Username Password Connection Type
+== Connection Types
+[[sfdc-config_basic]]
+=== Username Password (Deprecated)
 
-=== Parameters
+
+==== Parameters
 
 [%header%autowidth.spread]
 |===
 | Name | Type | Description | Default Value | Required
-| Username a| String |Username used to initialize the session. | |x
-| Password a| String |Password used to authenticate the user. | |x
-| Security Token a| String |User's security token. It can be omitted if your IP has been white listed on Salesforce | |
-| Authorization URL a| String |Web service URL responsible for user authentication. This is the URL for the endpoint that is configured to handle SOAP authentication requests. |`+https://login.salesforce.com/services/Soap/u/41.0+` |
-| Reconnection a| Reconnection |When the application is deployed, a connectivity test is performed on all connectors. If set to true, deployment fails if the test doesn't pass after exhausting the associated reconnection strategy. | |
-| Pooling Profile a| PoolingProfile |Characteristics of the connection pool | |
-| Read Timeout a| Number |Specifies the amount of time, in milliseconds, that the consumer waits for a response before it times out. Default value is 0, which means infinite. |0 |
-| Connection Timeout a| Number |Specifies the amount of time, in milliseconds, that the consumer attempts to establish a connection before it times out. Default value is 0, which means infinite. |0 |
-| Assignment Rule Id a| String | | |
-| Client Id a| String |Client ID for partners. | |
-| Time Object Store a| ObjectStore |An ObjectStore instance to use in SalesforceExtension#getUpdatedObjects. | |
```

Comment:
This was changed from "An ObjectStore instance to use in SalesforceExtension#getUpdatedObjects." to "An ObjectStore instance to use in SalesforceExtension.". I wonder if it meant that the instance should be used in an extension called getUpdatedObjects, and if the sentence should read "An ObjectStore instance to use in the SalesforceExtension getUpdatedObjects."

Reply:
(Here's I'm comparing to line 55 of the new file; lines 87 and 113 still have "SalesforceExtension#getUpdatedObjects".

Reply:
The original entry was:
| Time Object Store a| <<ObjectStore>> |  +++An ObjectStore instance to use in <a href="#_get_updated_objects">SalesforceExtension#getUpdatedObjects</a>
However there's no [[ObjectStore]] or [[_get_updated_objects]] anchors in this file for these links to link to. Similarly the Get Updated Objects operation has no ObjectStore reference. 

Changing this to:
An ObjectStore instance.